tests_profile

```diff
@@ -5,25 +5,26 @@
 # ¡Keep it MacOS/Ubuntu compatible!
 #
 # This profile contains:
-# - JAVA_HOME - path to currently selected JDK.
-#   jni-rs with enabled the invocation feature fails to build without JAVA_HOME.
+# - JAVA_HOME - path to currently selected JDK, required to build jni-rs
+#   with the invocation feature.
 # - JAVA_LIB_DIR - path to libjvm.
 # - RUST_LIB_DIR - path to Rust stdlib.
 # - RUST_COMPILER_VERSION - Rust version for building tests.
 # - RUSTFLAGS - Rust build options augmented with JAVA_LIB_DIR and RUST_LIB_DIR.
-#   That adds paths into the linker search paths where native code at runtime
-#   will look for dynamically loadable libraries.
-#   On Linux it is RUNPATH, so LD_LIBRARY_PATH is examined before.
-#   On MacOS it appends load commands to the end of the list, leaving @rpath/libNAME.dylib
-#   entries at the begin unchanged, so LD_LIBRARY_PATH also takes precedence.
-#   This affects all executables and dynamic libraries (application, tests, libjava_bindings).
+#   The linker will add paths to the libjvm and the Rust standard library
+#   to the runtime search paths. If you need to override the libjvm, set LD_LIBRARY_PATH
+#   to specify the path to it.
+#
+#   Cargo will set RUNPATH on Linux. On MacOS it appends load commands to the end
+#   of the list (¿which list), leaving @rpath/libNAME.dylib entries at the beginning unchanged.
```

Comment:
:warning:

Reply:
> which list

The list of mentioned here load commands.
https://github.com/aidansteele/osx-abi-macho-file-format-reference#load-command-data-structures

Reply:
TL;DR could you please explain how appending load commands corresponds to LD_LIBRARY_PATH and some entries at the beginning?

Reply:
At the beginning located default entries, looking like
```
load @rpath/libNAME.dylib
```
These commands is used to locate dylib in standard locations and with LD_LIBRARY_PATH, if it is defined, trying each location instead of the `@rpath` part.

Then with the `-rpath` link option, our custom load commands appear after these standard commands, and look like
```
load /at/some/path
```
The loader will try to execute load commands from the first to the last until one of them succeeds.